packages/core/tocco-ui/src/Button/Button.spec.js

```diff
@@ -1,97 +1,108 @@
+      testingLibrary.renderWithIntl(<Button disabled />)
+      jestExpect(screen.getByRole('button')).toBeDisabled()
+    })
 
-      test('should show pending spinner', () => {
-        let wrapper = enzymeUtil.mountEmbedded(<Button />)
-        expect(wrapper.find(LoadingSpinner)).to.have.length(0)
+    test('should not show pending spinner if pending is not set', async () => {
+      testingLibrary.renderWithIntl(<Button />)
+      const pendingIcon = await waitFor(() => screen.queryByTestId('icon-circle-notch'))
+      expect(pendingIcon).to.not.exist
+    })
 
-        wrapper = enzymeUtil.mountEmbedded(<Button pending={false} />)
-        expect(wrapper.find(LoadingSpinner)).to.have.length(0)
+    test('should not show pending spinner if pending is set to false', async () => {
+      testingLibrary.renderWithIntl(<Button pending={false} />)
+      const pendingIcon = await waitFor(() => screen.queryByTestId('icon-circle-notch'))
+      expect(pendingIcon).to.not.exist
+    })
 
-        wrapper = enzymeUtil.mountEmbedded(<Button pending />)
-        expect(wrapper.find(LoadingSpinner)).to.have.length(1)
-      })
+    test('should show pending spinner if pending is set', async () => {
+      testingLibrary.renderWithIntl(<Button pending />)
+      const pendingIcon = await waitFor(() => screen.getByTestId('icon-circle-notch'))
+      expect(pendingIcon).to.exist
+    })
 
-      test('should show icon', () => {
-        const wrapper = enzymeUtil.mountEmbedded(<Button icon="icon" />)
-        expect(wrapper.find(Icon)).to.have.length(1)
-      })
+    test('should show icon', async () => {
+      testingLibrary.renderWithIntl(<Button icon="cog" />)
+      const cogIcon = await waitFor(() => screen.getByTestId('icon-cog'))
+      expect(cogIcon).to.exist
+    })
 
-      test('should set default type to button', () => {
-        const wrapper = enzymeUtil.mountEmbedded(<Button />)
-        expect(wrapper.find('button').prop('type')).to.equal('button')
-      })
+    test('should set default type to button', () => {
+      testingLibrary.renderWithIntl(<Button />)
+      expect(screen.getByRole('button').getAttribute('type')).match(/button/)
+    })
 
-      test('should set type', () => {
-        const wrapper = enzymeUtil.mountEmbedded(<Button type="submit" />)
-        expect(wrapper.find('button').prop('type')).to.equal('submit')
-      })
+    test('should set type', () => {
+      testingLibrary.renderWithIntl(<Button type="submit" />)
+      expect(screen.getByRole('button').getAttribute('type')).match(/submit/)
     })
 
     describe('RouterLinkButton', () => {
       test('should show label', () => {
-        const wrapper = enzymeUtil.mountEmbedded(
+        testingLibrary.renderWithIntl(
           <MemoryRouter>
             <RouterLinkButton label="test" />
           </MemoryRouter>
         )
-        expect(wrapper.find('a').text()).to.equal('test')
+        expect(screen.getByText('test')).exist
```

Comment:
suggestion(non-blocking): since this is expected to be a link we could get by role 

```
screen.getByRole('link', { name: 'test' })
```